src/BcPrebidVast.js

```diff
@@ -636,11 +641,76 @@ function loadMolPlugin(callback) {
     }
 }
 
+function getAdRendererFromAdOptions(adOptions) {
+	// if adRenderer option is present use it for all ads
+	if (adOptions.hasOwnProperty('adRenderer') && adOptions.adRenderer &&
+		(adOptions.adRenderer === _rendererNames.MOL || adOptions.adRenderer === _rendererNames.IMA)) {
+		return {adRenderer: adOptions.adRenderer, needBreak: true};
```

Comment:
I would rename `needBreak` to `userSet` - it reflects more clearly _why_ the loop should break, rather than just the instruction that it should do so.

Reply:
Did it.